SU-HSDO/suhumsci
# Shs 105 color band

## Summary
This adds the color band 'light' variation to the color band component. 

## Steps to Test
1. Make sure you import the config and get the css compiled
2. Create a flexible page. Add a Color Band in the main region. You should see a 'Light' option in the background color. Choose that.
3. View the page and it should reflect the new `spotlight-lighten-8` color with the `hb-color-band--light` class.

## Screenshots
![Screen Shot 2022-11-01 at 10 04 24 AM](https://user-images.githubusercontent.com/8405274/199253194-5f39bff4-9971-4973-8203-61da12d5184f.png)

![Screen Shot 2022-11-01 at 9 59 15 AM](https://user-images.githubusercontent.com/8405274/199253210-dc289f65-5239-4e9f-909f-1c1c998fd7ad.png)
